strawberryfields/apps/graph/resize.py

```diff
@@ -308,6 +308,18 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import resize
+        >>> import networkx as nx
+        >>> graph = nx.wheel_graph(5)
+        >>> graph.remove_edge(0, 4)
+        >>> subgraph = [0, 1, 2]  # these nodes form a clique
+        >>> resize.clique_swap(subgraph, graph)
+        [0, 1, 4]
+
```

Comment:
```suggestion
        >>> graph = nx.lollipop_graph(5, 1)
        >>> graph.remove_edge(0, 4)
        >>> graph.remove_edge(0, 3)
        >>> subgraph = [0, 1, 2]
        >>> resize.clique_swap(subgraph, graph, node_select="degree")
        [1, 2, 4]

```
This would give also show an example use of the `node_select` argument.